### ISSUE TITLE
No submodule mapping found in .gitmodules for path 'merkle-patricia-proof'

### ISSUE THREAD

**Issue by khotkevych** [loiluu/peacerelay#2]
Failed to load "merkle-patricia-proof" submodule. 

Git clone command has the next output: 
`git clone --recursive https://github.com/loiluu/peacerelay.git
Cloning into 'peacerelay'...
remote: Counting objects: 426, done.
remote: Total 426 (delta 0), reused 0 (delta 0), pack-reused 426
Receiving objects: 100% (426/426), 6.79 MiB | 977.00 KiB/s, done.
Resolving deltas: 100% (251/251), done.
Checking connectivity... done.
fatal: no submodule mapping found in .gitmodules for path 'merkle-patricia-proof'
`

Or in the demo implementation the **txProof** is not supposed to be implemented?

And what is the best way to get **Rlp header** needed for **PeaceRelay** contract?

**Comment by igorbarinov** [comment]
@khotkevych , I have the same problem.  Did you manage to find `merkle-patricia-proof`. I have an idea that we can use Zac Mitton's  https://github.com/zmitton/eth-proof/blob/master/verifyProof.js 
and instead of getTxProof use eP.getTransactionProof(txHash).then((prf)=>{})
@naterush am I right? Thank you, guys.

**Comment by igorbarinov** [comment]
@loiluu can you clarify the issue with `merkle-patricia-proof` when you'll have time?

**Comment by naterush** [comment]
@igorbarinov @khotkevych yep - we used a minorly modified version of Zac Mitton's library to generate the proofs, etc. I can't find locate our changes currently - but it was simple stuff mostly just decoding or encoding. 

I would check out the Rinkeby Relay stuff here: https://github.com/ConsenSys/rb-relay. They made some progress on on-chain proof verification as well.
